plugin/completion/base_complete.py

```diff
@@ -135,7 +135,7 @@ def init(self, view, settings):
             cmake_prefix_paths=settings.cmake_prefix_paths,
             search_scope=search_scope)
 
-    def complete(self, view, cursor_pos, show_errors):
+    def complete(self, view, cursor_pos, current_job_id):
```

Comment:
Outdated documentation for new argument.
(GitHub won't let me comment on the documentation itself...)